.internal-ci/docker/entrypoints/fog-ledger.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+set -eo pipefail
+
+data="/fog-data"
+
+if [[ -n "${MC_LEDGER_DB_URL}" ]]
```

Comment:
:shrug: I generally lean on the 3 strike rule. first implementation is needed, second instance of the same implementation you may not know what's the same, third instance it's probably duplicate.

The current implementation may just be the nature of bash and docker, but I wonder if there should be a function(s) defined in a common bash script that the entrypoints call into. `restore_ledger_db()` and `restore_watcher_db()`.

IDK know this needs to change mainly just thinking/asking out loud

Reply:
I agree with the general principal, but each one of these entrypoints is for a separate instance that gets installed in a separate container. There's also the difference of how mobilecoind and the storage is implemented in each instance. So creating a standardized module (sourced file) is a much bigger project.

The whole set of containers and charts needs a refactor to clean up, standardize and utilize lessons learned.  

Also each time I try to set up abstractions for stuff like this, I get complaints about complexity... so its hard to win this one :shrug: